pkgs/development/python-modules/python-dbusmock/default.nix

```diff
@@ -1,15 +1,17 @@
-{ lib, buildPythonPackage, fetchPypi, runtimeShell,
+{ lib, buildPythonPackage, fetchFromGitHub, runtimeShell,
   nose, dbus, dbus-python, pygobject3,
   which, pyflakes, pycodestyle, bluez, networkmanager
 }:
 
 buildPythonPackage rec {
   pname = "python-dbusmock";
-  version = "0.18.3";
+  version = "0.19";
 
-  src = fetchPypi {
```

Comment:
Could you open issue against upstream that the release is missing from PyPI?

Reply:
https://github.com/martinpitt/python-dbusmock/issues/58

Reply:
Thanks @jonringer